src/io/image_iter_common.h

```diff
@@ -196,13 +199,13 @@ struct ImageNormalizeParam :  public dmlc::Parameter<ImageNormalizeParam> {
   // declare parameters
   DMLC_DECLARE_PARAMETER(ImageNormalizeParam) {
     DMLC_DECLARE_FIELD(seed).set_default(0)
-        .describe("The Random Seed.");
+        .describe("The random seed.");
     DMLC_DECLARE_FIELD(mirror).set_default(false)
-        .describe("If or not mirror the image.");
+        .describe("Whether to mirror the image or not.");
     DMLC_DECLARE_FIELD(rand_mirror).set_default(false)
-        .describe("If or not randomly the image.");
+        .describe("Randomly chose whether to mirror an image or not.");
```

Comment:
`If true, 50% of the images will be randomly mirrored (flipped along the horizontal axis)`